packages/core/utils/src/errorUtils.js

```diff
@@ -27,5 +27,14 @@ function jsonToError(json) {
   }
 }
 
+function symlinkPrivilegeWarning() {
```

Comment:
As you know, `parcel-bundler/test/utils.js` and `integration-tests/test/utils.js` are exactly same. Because of that, I wondered what to do. 

Now I found the `test-util` package. How about moving this to that package?

Reply:
Probably a good idea, didn't know there was a test-utils package already

Reply:
Just a question: 

How about merging `parcel-bundler/test/utils.js` and `integration-tests/test/utils.js` to `test-utils/utils.js` this time?